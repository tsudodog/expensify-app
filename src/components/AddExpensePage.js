import React from 'react';

const AddExpensePage = () => (
    <div>
        This is from my addExpense Component
    </div>
)

export default AddExpensePage;