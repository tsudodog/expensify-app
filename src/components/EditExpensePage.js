import React from 'react';

const EditExpensePage = (props) => {
    console.log(props);
    return (
        <div>
            This is from my EditExpensePage Component {props.match.params.id}
        </div>
    )
}

export default EditExpensePage;