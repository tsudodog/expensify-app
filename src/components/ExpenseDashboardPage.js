import React from 'react';
import ExpenseList from './ExpenseList'
import ExpenseListFilters from './ExpenseListFilters'
import ExpensesSummary from './ExpensesSummary'

const ExpenseDashboardPage = () => (
    <div>
        <ExpenseListFilters/>
        <ExpenseList></ExpenseList>
        <ExpensesSummary></ExpensesSummary>
    </div>
)

export default ExpenseDashboardPage