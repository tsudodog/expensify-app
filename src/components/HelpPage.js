import React from 'react';
const HelpPage = () => (
    <div>
        This is from my HelpPage Component
    </div>
)

export default HelpPage;