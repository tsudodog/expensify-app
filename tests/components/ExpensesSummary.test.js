import React from 'react'
import { shallow } from 'enzyme'
import {ExpensesSummary} from '../../src/components/ExpensesSummary'



test('is should correctly render expenses summary with 1 expense', ()=>{
  const wrapper = shallow(<ExpensesSummary expenseCount = {1} expsnesTotal = {235}/>)
  expect(wrapper).toMatchSnapshot()
})

test('should correctly render ExpensesSummary with more than one expense', () => {
    const wrapper = shallow(<ExpensesSummary expenseCount = {2} expsnesTotal = {500}/>)
    expect(wrapper).toMatchSnapshot()
})