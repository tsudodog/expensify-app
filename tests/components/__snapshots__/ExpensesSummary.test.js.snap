// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`is should correctly render expenses summary with 1 expense 1`] = `
<div>
  <h1>
    Viewing 
    1
     
    expense
     totalling 
    $0.00
  </h1>
</div>
`;

exports[`should correctly render ExpensesSummary with more than one expense 1`] = `
<div>
  <h1>
    Viewing 
    2
     
    expenses
     totalling 
    $0.00
  </h1>
</div>
`;
